src/ledgerwriter/src/main/java/anthos/samples/financedemo/ledgerwriter/LedgerWriterController.java

```diff
@@ -120,92 +129,57 @@ public ResponseEntity version() {
             bearerToken = bearerToken.split("Bearer ")[1];
         }
         try {
+            if (bearerToken == null) {
+                throw new IllegalArgumentException(
+                        EXCEPTION_MESSAGE_WHEN_AUTHORIZATION_HEADER_NULL);
+            }
             final DecodedJWT jwt = this.verifier.verify(bearerToken);
             // validate transaction
-            validateTransaction(jwt.getClaim("acct").asString(), transaction);
-
+            transactionValidator.validateTransaction(localRoutingNum,
+                    jwt.getClaim(CLAIM).asString(), transaction);
```

Comment:
From your doc and playing around with the code I think Claim is `Claim` type instead of string? (https://www.javadoc.io/doc/com.auth0/java-jwt/3.3.0/com/auth0/jwt/interfaces/Claim.html)